### PR TITLE
Add financial instruments to response

### PR DESCRIPTION
**Pull request by danielcopaciu**
Added financial instruments to organisation-api response.

Also, I added java memory limit. Without this circle ci build was failing due to large neo memory allocation.

### CODE REVIEW

File: organisations/cypher.go [hunk 1]
```diff
@@ -111,8 +117,12 @@ func (pcw CypherDriver) Read(uuid string) (organisation Organisation, found bool
  		{ id:parent.uuid, types:labels(parent), prefLabel:parent.prefLabel} as parent,
  		{ id:sub.uuid, types:labels(sub), prefLabel:sub.prefLabel, annCount:COUNT(ms) } as sub ORDER BY sub.annCount DESC
  		WITH o, pm, ind, lei, parent, collect(sub) as sub
-		WITH pm, ind, parent, sub, lei, { id:o.uuid, types:labels(o), prefLabel:o.prefLabel, labels:o.aliases} as o
- 		return collect ({o:o, lei:lei, parent:parent, ind:ind, sub:sub, pm:pm}) as rs
+ 		OPTIONAL MATCH (fi:FinancialInstrument)-[:ISSUED_BY]->(o)
+ 		OPTIONAL MATCH (fi)<-[:IDENTIFIES]-(figi:FIGIIdentifier)
+ 		WITH o, pm, ind, lei, parent, sub,
```
Comment (Nesze) [inline organisations/cypher.go]: (NeoRookie:) what is this for?

**Comment by Nesze** [comment]
some test?

**Comment by danielcopaciu** [comment]
Added test for financial instruments and updated example.json

**Comment by Nesze** [comment]
Lol, I can review at as many times I want

**Pull request merged**